packages/react/index.js

```diff
@@ -59,6 +59,7 @@ export {
   createMutableSource,
   Fragment,
   Profiler,
+  DebugTracingMode,
```

Comment:
Does it become enumerable (but undefined) export? If yes, it should be `unstable_`.

Reply:
This will never be undefined. It's just not re-exported by the .stable index. (This index file itself is never used directly.)